ci/build-all-steps.yml

```diff
@@ -43,6 +43,12 @@ steps:
     node index.js
   displayName: Verify all min agent demands are valid
 
+# TODO remove -- setting task_pattern directly
```

Comment:
Do we need to remove this?

Reply:
Yes! I had to do this to force it to build a couple tasks.